common/models/types/index.d.ts

```diff
@@ -90,6 +92,33 @@ declare interface LexicalModel {
    */
   configure(capabilities: Capabilities): Configuration;
 
+  /**
+   * Indicates that the language represented by the lexical model has syntactic casing
+   * behaviors.  Setting this to true will allow the predictive text engine to
+   * perform casing-based corrections and predictions.
+   * 
+   * If set to `false`, the default behavior for the `toKey` method will not perform
+   * casing modifications and will thus yield case-sensitive results.  This will not occur
+   * if `undefined` for backward-compatibility reasons.
```

Comment:
This is honsetly so difficult for me to parse. So I'm assuming if `languageUsesCasing` is false or undefined, that's the old behaviour (no automatic casing); if true, SOMETHING will apply automatic casing, correct? By using the provided `applyCasing()`? Or can `applyCasing` be undefined?

Reply:
Custom models may assume that a default `applyCasing` rule will be provided to serve as that "SOMETHING".